dumper/app/src/main/java/com/google/edwmigration/dumper/application/dumper/ConnectorArguments.java

```diff
@@ -884,14 +884,12 @@ private static ImmutableMap<String, String> buildDefinitionMap(
         resultMap.put(name, value);
         if (connector != null && propertyNamesByConnector.get(connector).contains(name)) {
           LOG.info("Parsed property: name='{}', value='{}'", name, value);
-        } else if (allPropertyNames.contains(name)) {
```

Comment:
I would suggest keeping the distinction between the unknown property and the property used in the wrong connector.

Reply:
Here is my thought process - The person that's using the dumper (with very high probability) has only one dialect to think about, e.g. - Teradata OR Redshift OR .... For him/her, it doesn't matter if the flag is correct for some other dialect and the message might force him/her to think about that. It feels more user friendly to just say "this is not a valid flag you can use". wdyt?

Reply:
> The person that's using the dumper (with very high probability) has only one dialect to think about
It's true, but even for one dialect we can have two connectors - for example, for Teradata we have a separate connector for metadata and for query logs. Some flags are compatible only with one of these connectors for the same dialect.

> It feels more user friendly to just say "this is not a valid flag you can use". 
The current implementation prints something different - "Property: name='%s', value='%s' is not compatible with connector '%s'". The user may think: "What are the implications? Is this property compatible with another connector? Or is it invalid altogether?"

I agree that for the first time user the message might not be that important, but for someone who runs the dumper a few times, it may be important, whether they've made a typo in the flag name or whether they provided a correct flag name, but used a wrong connector. Therefore I think that we should distinguish between "Unknown property" (or "Unrecognized property") and the known property, but associated with another connector.